doc/source/examples/analysis/alignment_and_rms/README.rst

```diff
@@ -10,10 +10,10 @@ The :mod:`MDAnalysis.analysis.align` and :mod:`MDAnalysis.analysis.rms`
 modules contain the functions used for aligning structures,
 aligning trajectories, and calculating root mean squared quantities.
 
-Demonstrations of alignment are in `align_structure`_,
-`align_trajectory_first`_, and `align_trajectory`_. Another example of
+Demonstrations of alignment are in `align_structure <aligning_structure_to_another.ipynb>`__,
+`align_trajectory_first <aligning_trajectory_to_frame.ipynb>`__, and `align_trajectory <aligning_trajectory.ipynb>`__. Another example of
```

Comment:
check links on https://mdanalysisuserguide--394.org.readthedocs.build/en/394/examples/analysis/alignment_and_rms/README.html ... not all of them work

Reply:
I just checked all the links here - https://mdanalysisuserguide--394.org.readthedocs.build/en/394/examples/analysis/alignment_and_rms/README.html and found they are working. Can you please check again?

Reply:
[align_trajectory_first](https://mdanalysisuserguide--394.org.readthedocs.build/en/394/examples/analysis/alignment_and_rms/aligning_trajectory_to_first_frame.ipynb) is 404

Reply:
> [align_trajectory_first](https://mdanalysisuserguide--394.org.readthedocs.build/en/394/examples/analysis/alignment_and_rms/aligning_trajectory_to_first_frame.ipynb) is 404

So I am getting this link after clicking on `aline_trajectory_first` which is working - https://mdanalysisuserguide--394.org.readthedocs.build/en/394/examples/analysis/alignment_and_rms/aligning_trajectory_to_frame.html. 
I also tried different browser, and its working here.

Reply:
The dead link is

> Demonstrations of alignment are in [align_structure](https://mdanalysisuserguide--394.org.readthedocs.build/en/394/examples/analysis/alignment_and_rms/aligning_structure_to_another.html), [align_trajectory_first](https://mdanalysisuserguide--394.org.readthedocs.build/en/394/examples/analysis/alignment_and_rms/aligning_trajectory_to_first_frame.ipynb), and [align_trajectory](https://mdanalysisuserguide--394.org.readthedocs.build/en/394/examples/analysis/alignment_and_rms/aligning_trajectory.html). Another example of generating an average structure from an alignment is demonstrated in [rmsf](https://mdanalysisuserguide--394.org.readthedocs.build/en/394/examples/analysis/alignment_and_rms/rmsf.html). Typically, trajectories need to be aligned for RMSD and RMSF values to make sense.

Reply:
... but after hard-refreshing I now get the working link!

Ok, so this is good now. Thank you for telling me your side.